opencraft/settings.py

```diff
@@ -77,7 +77,8 @@
     'pr_watch',
     'userprofile',
     'registration',
-    'backup_swift'
+    'backup_swift',
+    'community',
```

Comment:
I'm not sure about the name "community". That creates a divide between "us" on one side, and the "community" on the other side. 

I'm also not sure we need a full dedicated app for this. Why not use a unit test, like I was suggesting? It could go in the test suite of the instance app, since this is about instance configuration -- and you might even be able to make a useful test from this, by verifying some of the output.